app/controllers/surveys.rb

```diff
@@ -2,11 +2,46 @@
 require 'sinatra/base' # note this has changed from just 'sinatra'
 require 'sinatra/activerecord'
 require 'sinatra/contrib'
+require 'pry'
 
 class SurveysApp < Sinatra::Base
   set :root, File.expand_path("..",__dir__)
 
   get '/' do
     erb :dashboard
   end
+
+  get '/api/v1/questions' do
+    questions = Question.all
+    questions.to_json
+  end
+
+  get '/api/v1/responses' do
+    responses = Response.get_responses
+    responses.to_json
+  end
+
+  get '/api/v1/q_and_a' do
+    question_id = params[:question_id]
+    q_and_a = Question.get_q_and_a(question_id)
+    q_and_a.to_json
+  end
+
+  post '/api/v1/responses' do
+    response_data = params
```

Comment:
Does this line collect info from both the headers and params?

I'm unfamiliar with the `sinatra` way this gets done.

Reply:
forgive me, that is sudo code, I have not yet gotten back to the `post`.